src/chrome/content/rules/Sony.com.au.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	For other Sony coverage, see
+		+ Sony.xml
+
+	Nonfunctional hosts in *.sony.com.au:
+		- sony.com.au (r)
```

Comment:
You don't need to spell the domain name out since you already specified it's a subdomain of `sony.com.au`.